docs/cosri_v1.md

```diff
@@ -0,0 +1,79 @@
+
+# Smart On Fire /launch event:
+```json
+{
+    "version": "1",
+    "name": "sof_wrapper",
+    "tags": ["launch"],
+    "asctime": "2021-09-14 17:52:31,516+00:00",
+    "clinical-site": "Demo",
+    "deployment": "demo",
+    "system-name": "http://backend.cosri-demo.cirg.washington.edu/",
+    "message": "launch",
+    "subject": "Patient/1",
+    "levelname": "INFO"
+}
+```
+
+# PDMP event: patient lookup success
+```json
+{
+    "version": "1",
+    "name": "patientsearch",
+    "tags": ["search"],
+    "user": {
+        "DEA": "testDEAvalue",
+	"username": "test"
+    },
+    "asctime": "2021-09-14 18:01:50,224+00:00",
+    "clinical-site": "Demo",
+    "deployment": "demo",
+    "system-name": "http://pdmp.cosri-demo.cirg.washington.edu/",
+    "message": "PDMP found match",
+    "patient": {
+        "subject.id": "2",
+	"subject:Patient.birthdate": "eq1974-09-01",
+	"subject:Patient.name.given": "Harry",
+	"subject:Patient.name.family": "Osborn"
+    },
```

Comment:
Should this key be "subject" as used elsewhere instead of "patient"?

Reply:
that example is coming straight from the FHIR `Patient` resource, but I like your thinking - will see if there are other places we can work in `subject` more consistently.